src/Tests/ValidUntilTest.cs

```diff
@@ -11,19 +12,19 @@
 abstract class ValidUntilTest
 {
     [Test]
-    public void ValidUntil_is_correctly_set()
+    public async Task ValidUntil_is_correctly_set()
     {
         var cloudBlob = StubACloudBlob();
 
         SetValidUntil(cloudBlob, TimeSpan.FromHours(1));
-        var resultValidUntil = BlobStorageDataBus.GetValidUntil(cloudBlob);
+        var resultValidUntil = await BlobStorageDataBus.GetValidUntil(cloudBlob).ConfigureAwait(false);
```

Comment:
I thought we didn't care for `.configureawayt(false)` in tests.

Reply:
Tell that to the analyzer we are shipping that marks it as an error. AFAIK it doesn't hurt.

Reply:
Is anazlyzer part of the test project? Then it's carried over from the upstream project. Any way to exclude it?